modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java

```diff
@@ -194,7 +194,7 @@ public IgniteToKafkaCdcStreamer(
         if (!futs.isEmpty()) {
             try {
                 for (Future<RecordMetadata> fut : futs)
-                    fut.get(DFLT_REQ_TIMEOUT, TimeUnit.SECONDS);
+                    fut.get(kafkaReqTimeout, TimeUnit.SECONDS);
```

Comment:
Let's measure this timeout in milliseconds

Reply:
Done.